freqtrade/persistence/models.py

```diff
@@ -361,8 +454,11 @@ def adjust_min_max_rates(self, current_price: float) -> None:
 
     def _set_new_stoploss(self, new_loss: float, stoploss: float):
         """Assign new stop value"""
-        self.stop_loss = new_loss
-        self.stop_loss_pct = -1 * abs(stoploss)
+        self.set_stop_loss(new_loss)
```

Comment:
I think this calling-logic is dangerous
we now have 3 "set stoploss" methods that all work slightly differently - this should definitely be "somehow" simplified and unified to avoid confusion.